omod/src/main/resources/liquibase.xml

```diff
@@ -195,5 +195,31 @@
 							 tableName="metadatamapping_metadata_source"
 							 columnNames="name" />
 	</changeSet>
-
+	
+	<changeSet id="metadatamapping-2016-01-06-0800" author="jasonvena">
+		<preConditions onFail="MARK_RAN">
```

Comment:
I suggest you add a sort of "disclaimer" comment here. Something like: "Liquibase does not have a precondition for checking 'not null'. Instead, we follow the openmrs convention of just checking if the column itself exists."